frontend/src/components/profile-info.tsx

```diff
@@ -73,6 +81,52 @@ const ProfileInfo = () => {
         }
     }, [user, signedIn, setValue]);
 
+    useEffect(() => {
+        const fetchUserRegistrationSlugs = async () => {
+            if (!user?.email || !idToken) return;
+
+            const email = user.email;
+            const res = await RegistrationAPI.getUserRegistrations(email, idToken);
+            if (isErrorMessage(res)) {
+                toast({
+                    title: isNorwegian ? 'Det har skjedd en feil' : 'Something went wrong',
+                    description: res.message,
+                    status: 'error',
+                    duration: 8000,
+                    isClosable: true,
+                });
```

Comment:
Personlig synes jeg det er best med en feilmelding der arrangementene skal vises, i stedet for en toast.